src/main/java/com/kakao/cafe/common/exception/NotFoundDomainException.java

```diff
@@ -0,0 +1,11 @@
+package com.kakao.cafe.common.exception;
+
+public class NotFoundDomainException extends RuntimeException{
```

Comment:
```suggestion
public class DomainNotFoundException extends RuntimeException{
```
예외도 직접 만들었군요. 기존의 예외을 사용하는 것에 비해 어떤 장점이 있을까요?

Reply:
DB 조회 결과가 없을 경우 특이적 예외 처리를 하고자 사용했습니다.
findUser 의 경우 optional 결과를 받는데요. get()의 결과를 확실히 보장하기 위해서 
조회 결과 데이터가 없을 경우에 대해 예외를 사용하게 했습니다.